v2/lua/openresty_org/controller.lua

```diff
@@ -55,7 +71,33 @@ function _M.run()
         return ngx.redirect(uri .. "/", 301)
     end
 
-    local m, err = re_match(uri, [[ ^ / ( [a-z]{2} ) / (?: ([-\w]+) \.html )? $ ]], 'jox', nil, match_table)
+    local m = re_match(uri, [[ ^ /yum/ ( cn/ )? ( [a-z-0-9]+ ) /OpenResty.repo $ ]], 'jox', nil, match_table)
```

Comment:
`[a-z-0-9]` should better be `\w[-\w]+`.

`/OpenResty.repo` should be `/OpenResty\.repo` since `.` is special.